### PR TITLE
Fix not actually checking app healthchecks

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/ILpSOFhq/201-enable-cd-for-quick-win-and-possible-quick-win-apps

Previously in scenarios requesting the /healthcheck endpoint we only
verified the response code was 200, but this will always be the case
even if the app is unhealthy. This adjusts the /healthcheck scenarios
to actually verify the endpoint reports an "OK" status.

## Testing

**You should manually test your PR before merging.**

This app doesn't have CI setup, and it would be misleading to do so:
the behaviour of the tests changes depending on the environment they
are run in. You should manually test in applicable environments,
until you are confident your change is not a breaking one.

Example steps to test in Integration:

- Click "Configure" for the [Smokey job][]
- Change the "Branch specifier" to the name of your branch
- Run a new build of the Smokey project and check the results

## Deployment

**You need to manually deploy your PR after merging.**

Steps to deploy a change:

- Run the [Smokey deploy job][] to deploy the [continuous Smokey loop][]
- Repeat this in Staging and Production

> The manual [Smokey job][] will pick up changes on `master` automatically. You only need to do a manual deployment for the [continuous Smokey loop][].

[Smokey job]: https://deploy.integration.publishing.service.gov.uk/job/Smokey/
[continuous Smokey loop]: https://github.com/alphagov/govuk-puppet/blob/master/modules/monitoring/templates/smokey-loop.conf
[Smokey deploy job]: https://deploy.integration.publishing.service.gov.uk/job/Smokey_Deploy/

### CODE REVIEW

**Comment by benthorner** [comment]
Need to fix a problem with the JSON endpoint for finder-frontend not being exposed.

**Comment by benthorner** [comment]
Depends on outcome of: https://github.com/alphagov/govuk-puppet/pull/10852.

File: features/collections.feature [hunk 1]
```diff
@@ -71,4 +71,5 @@ Feature: Collections
   Scenario: Healthcheck
     Given I am testing "collections" internally
     When I request "/healthcheck"
-    Then I should get a 200 status code
+    Then JSON is returned
+    And I should see ""status":"ok""
```
Comment (thomasleese) [inline features/collections.feature:75]: The double `""` is a little bit jarring to see, but I think it's fine.

**Review by thomasleese — APPROVED** [review]

**Comment by benthorner** [comment]
Tested and works on Staging ✅. Just waiting for the prerequisite Puppet change ☝️ to roll out in Production.

**Pull request merged**